mod/groups/start.php

```diff
@@ -16,7 +16,8 @@
 function groups_init() {
 
 	elgg_register_library('elgg:groups', __DIR__ . '/lib/groups.php');
-
+	elgg_load_library('elgg:groups');
```

Comment:
but that would be a breaking chance as someone else could also require the library